hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileSystemUndeleteTest.java

```diff
@@ -21,19 +21,23 @@
 import static com.hedera.hapi.node.base.ResponseCodeEnum.UNAUTHORIZED;
 import static com.hedera.node.app.spi.fixtures.Assertions.assertThrowsPreCheck;
 import static com.hedera.test.utils.KeyUtils.A_COMPLEX_KEY;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
```

Comment:
Favour AssertJ over Junit Jupiter Assertions

Reply:
I agree with Thomas better to use AssertJ